src/components/rules-creation-page/steps/SaveStep.jsx

```diff
@@ -107,7 +109,12 @@ const SaveStep = ({ d2, syncRule, classes, onCancel, snackbar }) => {
 
                 <LiEntry label={i18n.t("Scheduling enabled")} value={syncRule.enabled.toString()} />
 
-                <LiEntry label={i18n.t("Frequency")} value={syncRule.frequency} />
+                {isValidCronExpression(syncRule.frequency) && (
+                    <LiEntry
+                        label={i18n.t("Frequency")}
+                        value={`${cronstrue.toString(syncRule.frequency)} (${syncRule.frequency})`}
```

Comment:
This snippet seens quite generic and it's repeated in the code. Maybe we can move it as a method of `SyncRule`?

Reply:
See https://github.com/EyeSeeTea/metadata-synchronization/pull/134/commits/d4eebaf7a6ac2897e547c59cf1986b291d022208